backend/api/views/computetask.py

```diff
@@ -91,6 +92,7 @@ def _register_in_orchestrator(tasks_data, channel_name):
         if "__tag__" in orc_task["metadata"]:
             raise Exception('"__tag__" cannot be used as a metadata key')
```

Comment:
This becomes redundant with the `__` constraint

Reply:
Not if we don't rename it, which seems not necessary since the orchestrator can deal with __ (that's why the metadata check is done before the backend adds this` __tag__`). Should we rename it to _tag_ still ? The backend will have to deal with both `__tag__` and `_tag_` (assets before this change and after).

Reply:
I'd leave it as is since the backend won't have to deal with before/after value.
Also, maybe move the `__tag__` in a constant to get a better visibility over setters/getters.

Then I still think this check is unnecessary, event without renaming as `orc_task["metadata"] == task_data.get("metadata") or {}`, this comes from the `task_data` (line 89) which is checked line 93.
So this will reject any metadata containing `__`, including `__tag__`. Then the `__tag__` value is added *after* the check on line 96, which is fine.

Reply:
Oh yes my bad, I answered too quiclky, this is definitely redundant here.

Reply:
unnecessary check remove and `__tag__` moved to a constant